packages/collector/test/tracing/cloud/aws-sdk/v3/dynamodb/test.js

```diff
@@ -0,0 +1,227 @@
+          });
+        });
+      } else {
+        describe(`getting result with error: ${withError ? 'yes' : 'no'}`, () => {
+          availableOperations.forEach(operation => {
+            const requestMethod = getNextCallMethod();
+            it(`operation: ${operation}/${requestMethod}`, async () => {
+              const withErrorOption = withError ? '?withError=1' : '';
+              const apiPath = `/${operation}/${requestMethod}`;
+
+              const response = await appControls.sendRequest({
+                method: 'GET',
+                path: `${apiPath}${withErrorOption}`,
+                simple: withError === false
+              });
+
+              /**
+               * Table takes some time to be available, even though the callback gives a success message
+               */
+              if (operation === 'createTable') {
+                await checkTableExistence(tableName, true);
+              }
+              return verify(appControls, response, apiPath, operation, withError);
+            });
+          });
+        });
+      }
+    });
+
+    function verify(controls, response, apiPath, operation, withError) {
+      return retry(() => {
+        return agentControls.getSpans().then(spans => verifySpans(controls, spans, apiPath, operation, withError));
+      }, retryTime);
+    }
+
+    function verifySpans(controls, spans, apiPath, operation, withError) {
+      const httpEntry = verifyHttpRootEntry({ spans, apiPath, pid: String(controls.getPid()) });
+      verifyExitSpan({
+        spanName: 'dynamodb',
+        spans,
+        parent: httpEntry,
+        withError,
+        pid: String(controls.getPid()),
+        extraTests: [
+          span => expect(span.data.dynamodb.op).to.equal(operationsInfo[operation]),
+          span => {
+            let expected;
+
+            if (operation !== 'listTables') {
+              if (operation !== 'createTable' || !withError) {
+                expected = tableName;
+              }
+            }
+
+            expect(span.data.dynamodb.table).to.equal(expected);
+          }
+        ]
+      });
+
+      if (!withError) {
+        verifyHttpExit({ spans, parent: httpEntry, pid: String(controls.getPid()) });
+      }
+    }
+  });
+
+  describe('tracing disabled', () => {
+    this.timeout(config.getTestTimeout() * 2);
+
+    const appControls = new ProcessControls({
+      appPath: path.join(__dirname, 'app'),
+      port: 3215,
+      useGlobalAgent: true,
+      tracingEnabled: false,
+      env: {
+        AWS_DYNAMODB_TABLE_NAME: tableName
+      }
+    });
+
+    ProcessControls.setUpHooksWithRetryTime(retryTime, appControls);
+
+    describe('attempt to get result', () => {
+      availableOperations.slice(1).forEach(operation => {
+        const requestMethod = getNextCallMethod();
+        it(`should not trace (${operation}/${requestMethod})`, async () => {
+          await appControls.sendRequest({
```

Comment:
Maybe we can also capture the response and verify that is as expected here. WDYT?